### PR TITLE
rebase kube to 1.19 rc2

### PR DESCRIPTION
**Pull request by stlaz**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,20 +9,20 @@ require (
...
+	k8s.io/apimachinery v0.19.0-rc.2
+	k8s.io/apiserver v0.19.0-rc.2
+	k8s.io/client-go v0.19.0-rc.2
+	k8s.io/code-generator v0.19.0-rc.2
+	k8s.io/component-base v0.19.0-rc.2
+	k8s.io/kube-openapi v0.0.0-20200427153329-656914f816f9
 )
 
-replace k8s.io/apiserver => github.com/openshift/kubernetes-apiserver v0.0.0-20200403085848-ec5d03abc6b3
+replace k8s.io/apiserver => github.com/openshift/kubernetes-apiserver v0.0.0-20200727120649-a5e0645f4b46
```
Comment (p0lyn0mial) [inline go.mod:28]: looks, good, it targets https://github.com/openshift/kubernetes-apiserver/commits/openshift-apiserver-4.6-kubernetes-1.19.0-rc.2

File: go.mod [hunk 2]
```diff
@@ -9,20 +9,20 @@ require (
...
-	k8s.io/code-generator v0.18.0
-	k8s.io/component-base v0.18.0
-	k8s.io/kube-openapi v0.0.0-20200121204235-bf4fb3bd569c
+	k8s.io/api v0.19.0-rc.2
+	k8s.io/apimachinery v0.19.0-rc.2
+	k8s.io/apiserver v0.19.0-rc.2
+	k8s.io/client-go v0.19.0-rc.2
+	k8s.io/code-generator v0.19.0-rc.2
+	k8s.io/component-base v0.19.0-rc.2
+	k8s.io/kube-openapi v0.0.0-20200427153329-656914f816f9
```
Comment (p0lyn0mial) [inline go.mod:25]: okay, the same as in https://github.com/openshift/kubernetes/blob/master/go.mod#L130

**Comment by p0lyn0mial** [comment]
failing `ci/prow/verify` looks real

**Comment by stlaz** [comment]
I rebased this PR on top of https://github.com/openshift/oauth-apiserver/pull/15, that one should merge earlier anyway

**Pull request merged**